desktop-app/src/pages/SimulationInsights.jsx

```
@@ -0,0 +1,450 @@
+                const isSelected = selectedPatterns.includes(learning.id);
+
+                return (
+                  <div
+                    key={learning.id}
+                    onClick={() => togglePattern(learning.id)}
+                    className={`border-2 rounded-lg p-5 cursor-pointer transition-all ${
+                      isSelected
+                        ? 'border-blue-500 bg-blue-50'
+                        : 'border-gray-200 hover:border-gray-300'
+                    }`}
+                  >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)